### DISCUSSION TITLE
Uploaded images are no longer selected automatically when you add or change images

### DISCUSSION THREAD

**Discussion by JaniceManwiller** [GitbookIO/community#676] [0. General bugs and questions] [answered]
Previously, when you added a new image or chose to replace an existing image, when you uploaded the image to use, it was automatically selected for the image object that you were adding or updating. So the steps were:

1. Select the option to add or update an image.
2. Select the file from the file system.

That no longer happens. Now, after you upload the image, you always have to click the image to actually apply it to the image object.

1. Select the option to add or update an image.
2. Select file from file system.
3. Click the file again.

It's minor, but very inconvenient. If I select the option to update an image and then upload an image from the file system, then presumably that's the file that I want to use.

**Comment by SamyPesse** [comment, marked as answer]
Hi @JaniceManwiller Thank you for identifying this issue, this is indeed a regression caused by changes we shipped to the files manager. We'll get it fixed asap :)
